DependencyInjection/Configuration.php

```diff
@@ -28,6 +28,8 @@ public function getConfigTreeBuilder()
                     ->children()
                         ->arrayNode('email')
                             ->children()
+                                ->variableNode('from')
```

Comment:
what if I want to send messages from no-reply/info/accounts/support@gmail.com? No reason to keep only one. 
I think user can manage it himself.